src/libstd/keyword_docs.rs

```diff
@@ -710,3 +789,79 @@ mod loop_keyword { }
 /// [book]: ../book/ch05-01-defining-structs.html
 /// [reference]: ../reference/items/structs.html
 mod struct_keyword { }
+
+#[doc(keyword = "super")]
+//
+/// The parent of the current [module].
+///
+/// [module]: ../../nightly/book/ch07-02-defining-modules-to-control-scope-and-privacy.html
+mod super_keyword { }
+
+#[doc(keyword = "trait")]
+//
+/// A single interface shared across types.
+mod trait_keyword { }
+
+#[doc(keyword = "true")]
+//
+/// A value of type `bool` representing logical **true**.
+mod true_keyword { }
+
+#[doc(keyword = "type")]
+//
+/// Give a new name to an existing type.
+mod type_keyword { }
+
+#[doc(keyword = "unsafe")]
+//
+/// Opt out of statically enforced memory safety.
```

Comment:
Only true for `unsafe { .. }` and `unsafe impl` but not of `unsafe fn` and `unsafe trait`.
It's better to talk about that `unsafe { ... }` is a "trust me" construct and that `unsafe fn` has proof obligations.

Reply:
I don't think that the distinction between "unsafe to use" and "unsafe to implement" needs to be mentioned here, although the imperative does make it sound like I'm talking exclusively about `unsafe` blocks. This line was adapted from the chapter summary in the Rust book:

> All the code we’ve discussed so far has had Rust’s memory safety guarantees enforced at compile time. However, Rust has a second language hidden inside it that doesn’t enforce these memory safety guarantees: it’s called  unsafe Rust...

Reply:
> I don't think that the distinction between "unsafe to use" and "unsafe to implement" needs to be mentioned here,

That's not the distinction I'm going for since your current change covers both `unsafe impl` and `unsafe { ... }`. Rather, it is the distinction between `unsafe { ... }` and `unsafe fn`.

I think I might go for something more akin to:
```suggestion
/// Verify safety yourself or define an unsafe abstraction.
```

The point here is to tell the reader that you can't just do whatever you want in `unsafe { ... }` and that certain invariants the compiler expects still must be upheld.

Reply:
Another option, once again paraphrased from the book, but split into "interfaces" and "code" for distinguishing `unsafe fn`/`unsafe trait` versus `unsafe { ... }`

```suggestion
/// Code or interfaces whose memory safety cannot be guaranteed at compile time.
```

Reply:
I think `Verify` is more imperative, but I'd change your version to:
```suggestion
/// Code or interfaces whose safety cannot be verified by the type system.
```